layouts/_default/list.html

```diff
@@ -14,13 +14,13 @@
         {{ if eq .Type "posts" }}
         <li>
           {{ if .Draft }}
-          DRAFT: 
+          DRAFT:
           {{ end }}
           <a class="post-list" href="{{ .Permalink }}">{{ $page.Title }}</a>
 
           <p class="footnote">
-          <time datetime="{{ $page.Date.Format "01-02-2006" }}">
-            {{ $page.Date.Format "01-02-2006" }}
+          <time datetime="{{ $page.Date.Format .Site.Params.dateformat }}">
+            {{ $page.Date.Format .Site.Params.dateformat }}
```

Comment:
So I think the addition of a `Params` section for `dateformat` is great, but I think we can still do with a default one like so:

```
{{ if .Site.Params.dateformat }}
  {{ $page.Date.Format .Site.Params.dateformat }}
{{ else }}
  {{ $page.Date.Format "01-02-2006" }}
{{ end }}
```
It is unfortunately more verbose, but this is the cleanest way to go about it if we want a drop-in theme template without having to add anything extra on the `config` from the get-go.